File: leetcode/implementation/1672. Richest Customer Wealth/1672. Richest Customer Wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int maxWealth = 0;
        for(auto acc : accounts) {
            int wealth = 0;
            for(auto a : acc) {
                wealth += a;
            }
            maxWealth = max(maxWealth, wealth);
        }
        return maxWealth;
    }
};

File: leetcode/implementation/1672. Richest Customer Wealth/1672. Richest Customer Wealth.rs
use std::cmp::max;

impl Solution {
    pub fn maximum_wealth(accounts: Vec<Vec<i32>>) -> i32 {
        let mut maxWealth = 0;
        for i in 0..accounts.len() {
            let mut wealth = 0;
            for j in 0..accounts[i].len() {
                wealth += accounts[i][j];
            }
            maxWealth = max(maxWealth, wealth);
        }

        return maxWealth;
    }
}
